packages/backend-glr/src/Happy/Backend/GLR/CLI.hs

```diff
@@ -0,0 +1,58 @@
+module Happy.Backend.GLR.CLI(Flag(..), options, characteristicOption, hasCharacteristicFlag, parseFlags) where
+
+import Prelude hiding (filter)
+import Happy.Backend.GLR
+import System.Console.GetOpt
+
+-------- CLI flags and options --------
+
+data Flag =
+    OptGLR |
```

Comment:
@knothed actually I think this should be a top level flag. The top level decides what backend to to use. Once we are in this module we've already made that this, and this package doesn't use this flag at all.

Reply:
True, done.

I've also thought about this, but wasn't sure as it requires every executable that uses multiple backends to declare these backend-switching flags itself, leading to code duplication across executables.

But that shouldn't really be a problem.